fs/config/configfile/configfile.go

```diff
@@ -140,15 +159,15 @@ func (s *Storage) Save() error {
 	}
 
 	var fileMode os.FileMode = 0600
-	info, err := os.Stat(configPath)
+	info, err = os.Stat(configPathReal)
```

Comment:
I think I'd try not to rename the `configPath` variable to minimize the patch.

Reply:
The reason for this was to keep the following at the end, with the s.fi pointing to FileInfo retrieved from the original path. To reduce risk of introducing issue due to some nuances I hadn't considered. But I guess Stat follows the symlink anyway and we can trust the result to be the same directly on the resolved path..?

```
// Update s.fi with the newly written file
s.fi, _ = os.Stat(configPath)
```

Reply:
I guess the s.fi is only to record the modtime and size:

https://github.com/rclone/rclone/blob/243bcc9d0787276ccaccbbd914d70ede6a0305e6/fs/config/configfile/configfile.go#L40

So that should be safe - will fix.